### PR TITLE
fix wrong fbit config during tests

### PR DESCRIPTION
**Pull request by nicolasparada**
Fix invalid pipeline config used during tests.

### CODE REVIEW

File: client/pipeline_test.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ const testFbitConfigWithAddr3 = `[INPUT]
 	Port     5140
 	Mode     tcp
 [INPUT]
-	Name     syslog.1
```
Comment (nicolasparada) [inline client/pipeline_test.go:29]: @pwhelan can you confirm me that this was invalid?

**Review by pwhelan — APPROVED** [review]
LGTM.

**Comment by nicolasparada** [comment]
Uhhh, nice. We got a panic https://github.com/calyptia/api/runs/7961711404?check_suite_focus=true#step:7:172

File: client/pipeline_test.go (hunk 1 above)
Comment (edsiper) [reply client/pipeline_test.go:29]: that's invalid.

Comment (nicolasparada) [reply client/pipeline_test.go:29]: Thanks @edsiper 
We had some tests in Cloud back-end that were not too strict. I change it to be stricter and cause this one to fail.

**Pull request merged**